index.js

```diff
@@ -62,7 +86,106 @@ function createTaskCreatedAt(taskCreatedAt) {
 function createTaskCheckBox() {
   return `<input
             type="checkbox"
-            class="table-box-check-box"
             name="checkbox-done"
           /> `;
 }
+
+// function sortTasks(sortType) {
+//   if (sortType == "byTitle") {
+//     renderTasks(sortTasksByTitle());
+//   } else {
+//     renderTasks(unsortTasks());
+//   }
+// }
+
+// function sortTasksByTitle() {
+//   const collator = new Intl.Collator("en", {
+//     numeric: true,
+//   });
+
+//   return tasks.sort((randomTask1, randomTask2) =>
+//     collator.compare(
+//       randomTask1.title.toUpperCase(),
+//       randomTask2.title.toUpperCase()
+//     )
+//   );
+// }
+
+// function unsortTasks() {
+//   return tasks.sort((task1, task2) => (task1.id > task2.id ? 1 : -1));
+// }
+
+function filterTasksByTitle(text) {
+  renderTasks(tasks.filter((task) => task.title.includes(text)));
+}
+
+function filterTasksByTime(time) {
+  if (time == "") {
```

Comment:
Please use `===` instead `==` Or you can use `switch, case`. There should be an `else` for fallback of rendering all tasks.

Reply:
Oh such a big lesson for me,Thanks ^^